### ISSUE TITLE
hello is it possible to use bayesianoptimization at h2o.deeplearning??

### ISSUE THREAD

**Issue by sungreong** [yanyachen/rBayesianOptimization#38]
hi i want to use bayesianoptimization in h2o.deeplearning.
so i try this code. 
`initial_weight_distribution = c("UniformAdaptive","Uniform", "Normal") 
loss = c("Automatic", "CrossEntropy", "Huber" , "Absolute" , "Quadratic") 
activation = c("RectifierWithDropout" , "MaxoutWithDropout" ,"TanhWithDropout") 
acq <- c("ucb" , "ei" , "poi")
library(rlist)

source("./deep_source.R")
####
for ( init in initial_weight_distribution){
  for (loss_1 in loss){
    for( act in activation){
      for (acq_1 in acq){
        print(paste("-----",init ,loss_1 , act , acq_1,"-----",sep="-"))
        aa <- tryCatch( select(h2o_bayes , parameter) , error = function(e){ list(print(e)) } )
        time <- Sys.time()
        file_name <- paste(substr(time,3,4),substr(time,6,7),substr(time,9,10),substr(time,12,13)
                           ,init ,loss_1 , act , acq_1, sep="_")
        list.save(aa , file = paste0("./loop_para/",file_name,".rds") )
        print("==============================")
      }
    }
  }
}`

but An error occurs in the middle. " <simpleError: DistributedException from localhost/127.0.0.1:54321: '> "

thanks :)
